server/bin/www

```diff
@@ -1,25 +1,32 @@
 #!/usr/bin/env node
 
 /**
+ * Run database migrations
+ */
+// TODO add 'run database migrations'
+// console.log('Running migrations...');
+// db.migrate.latest();
+
```

Comment:
If we don't need this anymore, please delete it :) don't leave comments in the code on master branch.

Reply:
@bill-feng Thanks for noticing! Understand, removed it!:)